component.json

```diff
@@ -91,6 +91,41 @@
           }
         }
       }
+    },
+    "delete": {
+      "main": "./lib/actions/delete.js",
+      "title": "Delete files",
+      "description": "Delete file by name in provided package",
+      "fields": {
+        "directory": {
+          "viewClass": "TextFieldView",
+          "label": "Directory Name",
```

Comment:
How about replacing `Directory Name` with `Path`?

Reply:
Why - it's directory name?